_layouts/default.html

```diff
@@ -60,7 +60,7 @@ <h1 id="project_title">{{ site.title | default: site.github.repository_name }}</
                         <a href="https://www.swansea.ac.uk/"><img src="/assets/images/swansea-university-2017.lighter.en.png" alt="Swansea University" height="65"></a>
                     </td>
                     <td>
-                        <a href="https://www.exeter.ac.uk/"><img src="/assets/images/Uni_Exeter.inverse.png" alt="University of Exeter" height="60"></a>
+                        <a href="https://www.exeter.ac.uk/"><img src="/assets/images/University of Exeter_Crest_Logo_RGB_Uni_Landscape_Neg_Lrg.png" alt="University of Exeter" height="60"></a>
```

Comment:
Good spot to update the UoE logo for the rebranded ones.

Have you forgotten to git add these pngs?